apps/pie-docs/src/components/text-input/code/code.md

```diff
@@ -116,6 +120,25 @@ import { PieTextInput } from '@justeattakeaway/pie-text-input/dist/react';
 </PieTextInput>
 ```
 
+```jsx
+// React templates (using Next 13)
+import { PieTextInput } from '@justeattakeaway/pie-text-input/dist/react';
```

Comment:
```suggestion
import { PieTextInput } from '@justeattakeaway/pie-webc/react/text-input';
```

Reply:
Next 13 apps requiring SSR should import the components individually, shouldn't they?